Dockerfile-defaultworker

```diff
@@ -4,6 +4,7 @@ MAINTAINER Phuong Doan pdoan@cuahsi.org
 
 # inplaceedit in pip doesn't seem compatible with Django 1.11 yet...
 RUN pip install git+https://github.com/theromis/django-inplaceedit.git@e6fa12355defedf769a5f06edc8fc079a6e982ec
+RUN pip install markdown
```

Comment:
remove this and update hs_docker_base to release-1.9.3

Reply:
Fixed in 3e85239f2